src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -160,6 +160,8 @@ window.qBittorrent.DynamicTable ??= (() => {
             }, true);
 
             this.dynamicTableDiv.addEventListener("touchstart", (e) => {
+                if (e.target.classList.contains("selectRowCheckbox"))
+                    return; // ignore touch events on checkboxes, otherwise breaks on mobile
```

Comment:
```suggestion
                // ignore touch events on checkboxes, otherwise breaks on mobile
                if (e.target.classList.contains("selectRowCheckbox"))
                    return;

```